.travis.yml

```diff
@@ -7,6 +7,7 @@ matrix:
     - install:
       - env | grep TRAVIS > .env
       - env | grep CI >> .env
+      - echo "XDG_CACHE_HOME=/src/.cache" >> .env
```

Comment:
Why not put this in `travis-docker-compose.yml`? I'm fine either way but just wondering